angular/src/app/workflows/workflow-list-multi/workflow-list-multi.component.scss

```diff
@@ -1,2 +1,14 @@
 @import "../../../styles/modules/colors";
 
+::ng-deep mat-card-content {
```

Comment:
Does actually all of the `::ng-deep` needs to be there? I suggest putting everything under 
```
mat-card {
}
```
since all of them related